client/js/components/pages/TeamAddPage.js

```diff
@@ -15,9 +15,16 @@ class TeamAddPage extends Component {
 
     return (
       <DocumentTitle title="Add team | X-Map">
-        <div className="panel">
-          <TeamForm onSubmit={actions.teamCreate} onSuccess={this.redirectToTeamsPage.bind(this)} errors={errors}/>
-        </div>
+        <article id="TeamAddPage" className="page card">
+          <header className="card-header">
+            <h3 className="card-title">Add team</h3>
+            <p className="card-subtitle">Create new team then edit the team to add people.</p>
```

Comment:
"Create a new team"
I was also thinking about adding users to teams on team page, not in the edit page, as link/unlink operation is a separate API request - it doesn't happen during team update. Doing that in team edit page would be a bit misleading as changes to team members would be persisted even if you don't press the save button